crates/meilisearch/tests/search/mod.rs

```diff
@@ -135,13 +121,13 @@ async fn bug_5547() {
 #[actix_rt::test]
 async fn search_with_stop_word() {
     // related to https://github.com/meilisearch/meilisearch/issues/4984
-    let server = Server::new().await;
-    let index = server.index("test");
+    let server = Server::new_shared();
+    let index = server.unique_index();
 
     let (_, code) = index
         .update_settings(json!({"stopWords": ["the", "The", "a", "an", "to", "in", "of"]}))
         .await;
-    meili_snap::snapshot!(code, @"202 Accepted");
+    snapshot!(code, @"202 Accepted");
 
     let documents = DOCUMENTS.clone();
     index.add_documents(documents, None).await;
```

Comment:
This test is flaky and blocking the CI because right below this comment we're still waiting for the task `1` on a shared instance.
Instead, we should retrieve the actual id of the task 👍